content/en/_index.md

```diff
@@ -250,6 +339,12 @@ try (CloudClient dlp = CloudClient.create()) {
   // make a request with the client
 }
 {{< /tab >}}
+{{< tab header="Go" >}}
+// Don't initialize one client for the entire set of samples and pass it as an
+// argument. Each sample should initialize its own client/service.
```

Comment:
This comment should be targeted to the user and should try give guidance on how to use clients (and if there are any limits - e.g. thread safety), and if there is any clean up needed to remove background resources.

Reply:
Fixed.